### ISSUE TITLE
508-defect-4: Avoid desktop-centric instructions

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25015]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

Not all users will be able to "click" on accordions to open them. Consider rewriting language to be more inclusive of other mediums.

## Acceptance Criteria

- [ ] Desktop-centric instruction "click" is substituted with more inclusive copy

## Proposed Solution (if known)

Options as follows:
- Remove the instructions completely (should be intuitive what to do) OR
- Provide instructions for all mediums e.g. "Click, tap, or hit `space` on your keyboard for more details"
- Provide interaction-focused instructions e.g. "Open a topic for more details"

## Screenshots or Trace Logs

<img width="945" alt="Screen Shot 2021-05-20 at 11 13 56 AM" src="https://user-images.githubusercontent.com/14154792/119005308-5428f700-b95d-11eb-8edd-0e3cf1dac753.png">
